Geometry/HGCalCommonData/interface/HGCalGeomParameters.h

```diff
@@ -79,20 +77,20 @@ class HGCalGeomParameters {
   };
 
 private:
-  void loadGeometryHexagon(const std::map<int, HGCalGeomParameters::layerParameters>& layers,
-                           std::vector<HGCalParameters::hgtrform>& trforms,
-                           std::vector<bool>& trformUse,
-                           const std::unordered_map<int32_t, int32_t>& copies,
-                           const HGCalParameters::layer_map& copiesInLayers,
-                           const std::vector<int32_t>& wafer2copy,
-                           const std::vector<HGCalGeomParameters::cellParameters>& wafers,
-                           const std::map<int, int>& wafertype,
-                           const std::map<int, HGCalGeomParameters::cellParameters>& cellsf,
-                           const std::map<int, HGCalGeomParameters::cellParameters>& cellsc,
-                           HGCalParameters& php);
-  void loadGeometryHexagon8(const std::map<int, HGCalGeomParameters::layerParameters>& layers,
-                            std::map<std::pair<int, int>, HGCalParameters::hgtrform>& trforms,
-                            const int& firstLayer,
+  void loadGeometryHexagon(const std::map<int, HGCalGeomParameters::layerParameters>&,
```

Comment:
what is the purpose of this change?

Reply:
Header file need not know the name of the variable. It is the case for declaration of many methods. Otherwise this change does not add to anything

Reply:
It is nevertheless considered a good habit to name the parameters in the header for better self-documentation of the interface. Actually our code rule 4.27 also recommends that "Provide meaningful argument names in method declarations in the header ﬁle to indicate usage, unless the type fully describes the usage."

Reply:
Doing as suggested